include/onnxruntime/core/platform/EigenNonBlockingThreadPool.h

```diff
@@ -130,6 +130,12 @@
 namespace onnxruntime {
 namespace concurrency {
 
+#ifdef _WIN32
+using CHAR_TYPE = wchar_t;
```

Comment:
>CHAR_TYPE [](start = 6, length = 9)

see also ORTCHAR_T. why is it necessary to use different char types for the thread pool name?

Reply:
Would be good to tidy this up, but I think we have this in a few places going back to when we pulled in the original Eigen code.  If it's not an easy improvement as part of the current PR, we should do this as part of making the Eigen code more properly integrated in the ORT repo (e.g., we can avoid the need for additional sub-classing in ExtendedThreadPoolInterface).